lm/lm_image.py

```diff
@@ -21,7 +21,7 @@ def __init__(
         extra_config=None,
         extra_post_config=None,
         encoding="utf-8",
-        mem=2,
+        mem=12,
```

Comment:
are you sure we need this much by default (especially if it is configurable)? I think the memory requirement would be [Size of the LM] + [some MB for the Job].

So while I would agree that 2G are too low in most cases, 6 or 8 would be a more reasonable default value. (What is the current mem/core ratio in the cluster?)

Reply:
I had it crash w/ 8GB on LS960 so I went with 12GB.

Reply:
mh.. okay.. I'll leave this to the other users to decide which default they prefer.

Reply:
It's still [Size of LM] + [some MB for the Job], just that [some MB for the job] is very large ;)

Reply:
I would prefer if 8 works, but if even for a standard task like ls960 it is failing we should have 12.